AutorestSwift.xcodeproj/xcuserdata/sacheu.xcuserdatad/xcschemes/xcschememanagement.plist

```diff
@@ -12,12 +12,12 @@
 		<key>AutorestSwiftPackageDescription.xcscheme_^#shared#^_</key>
```

Comment:
This file shouldn't be under source control. I thought we added `xcuserdata/*` to the `.gitignore`?

Reply:
Yeah. I added 'AutorestSwift.xcodeproj/xcuserdata/'. to .gitigore. Not sure why git does not ignore this file.

Reply:
Revert the change.